static/app/views/starfish/views/screens/screenLoadSpans/sidebar.tsx

```diff
@@ -0,0 +1,166 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import omit from 'lodash/omit';
+
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {NewQuery} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {formatAbbreviatedNumber, getDuration} from 'sentry/utils/formatters';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {MiniAggregateWaterfall} from 'sentry/views/performance/browser/webVitals/components/miniAggregateWaterfall';
+import {SidebarSpacer} from 'sentry/views/performance/transactionSummary/utils';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import {YAxis} from 'sentry/views/starfish/views/screens';
+import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
+
+type ScreenMetrics = {
+  [key in YAxis]: number;
+};
+
+type Props = {
+  transaction: string;
+  screenMetrics?: ScreenMetrics;
+};
+
+export function ScreenLoadSpansSidebar({transaction}: Props) {
+  const {selection} = usePageFilters();
+  const location = useLocation();
+  const searchQuery = new MutableSearch([
+    'event.type:transaction',
+    'transaction.op:ui.load',
+    `transaction:${transaction}`,
+  ]);
+
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+
+  const queryStringPrimary = appendReleaseFilters(
+    searchQuery,
+    primaryRelease,
+    secondaryRelease
+  );
+
+  const newQuery: NewQuery = {
+    name: 'ScreenMetricsRibbon',
+    fields: [
+      `avg_if(measurements.time_to_initial_display,release,${primaryRelease})`,
+      `avg_if(measurements.time_to_initial_display,release,${secondaryRelease})`,
+      `avg_if(measurements.time_to_full_display,release,${primaryRelease})`,
+      `avg_if(measurements.time_to_full_display,release,${secondaryRelease})`,
+      'count()',
+    ],
+    query: queryStringPrimary,
+    dataset: DiscoverDatasets.METRICS,
+    version: 2,
+    projects: selection.projects,
+  };
+  const eventView = EventView.fromNewQueryWithLocation(newQuery, location);
+  const {data, isLoading} = useTableQuery({
+    eventView,
+    enabled: !isReleasesLoading,
+  });
+
+  const undefinedText = '--';
+
+  return (
+    <Fragment>
+      <SectionHeading>{t('Count')}</SectionHeading>
+      <SidebarMetricsValue>
+        {isLoading
+          ? undefinedText
+          : formatAbbreviatedNumber(data?.data[0]?.['count()'] as number)}
+      </SidebarMetricsValue>
+      <SidebarSpacer />
+      <SectionHeading>{t('Avg TTID (Release 1)')}</SectionHeading>
```

Comment:
I wonder if there's a better way to show release 1 vs. release 2 rather than stacking it like this. It would also be nice to get % change between releases somewhere here.

Maybe something like?

```
Avg TTID
Release 1       Release 2
200ms           225ms (+12.5%)
```

Should we also make release clickable? So you can go into releases view from it?

Reply:
Good suggestion! I will implement something as a follow up!

Reply:
tracking it [here](https://www.notion.so/sentry/Update-how-we-show-release-1-release-2-metrics-in-sidebar-b06daff2787045319b1b462a623b952e?pvs=4)